scripts/mkrepo.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# Take the deb and bundle it into a apt repository
+if [[ $# -lt 1 ]]
+then
+    echo "Usage $0 <config dir>"
+    exit
+fi
+
+confdir=$1
+
+set -x
+set -e
+
+ver=`node -e "require('fs'); console.log(JSON.parse(fs.readFileSync('package.json')).version)"`
```

Comment:
🤢🤮

NodeJS is great, but ohgod. Can we use something like `jq` instead? Stack Overflow says it is possible: https://stackoverflow.com/questions/53967693/how-to-run-jq-from-gitbash-in-windows

Reply:
maybe, but it's another thing everyone has to install, and we know we have node... (This is building the deb repo so this one we don't need to work in windows).

Reply:
it doesn't sound too painful to require another dependency (it's the same apt-get?)